akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingInternalsSpec.scala

```diff
@@ -39,5 +43,33 @@ class ClusterShardingInternalsSpec extends AkkaSpec("""akka.actor.provider = "cl
         ArgumentMatchers.eq(extractEntityId),
         ArgumentMatchers.eq(extractShardId))
     }
+
+    "HandOffStopper must stop the entity even if the entity doesn't handle handOffStopMessage" in {
+      case class HandOffStopMessage() extends NoSerializationVerificationNeeded
+      class EmptyHandlerActor extends Actor {
+        override def receive: Receive = {
+          case _ ⇒
+        }
+
+        override def postStop(): Unit = {
+          super.postStop()
+        }
+      }
+
+      val probe = TestProbe()
+      val shardName = "test"
+      val emptyHandlerActor = system.actorOf(Props(new EmptyHandlerActor))
+      val handOffStopper = system.actorOf(
+        Props(new HandOffStopper(shardName, probe.ref, Set(emptyHandlerActor), HandOffStopMessage, 5.seconds))
```

Comment:
to speed up this test you should decrease the timeout. I think it tests what it is supposed to also with 10.millis

Reply:
Done